cmd/stdiscosrv/database.go

```diff
@@ -352,3 +356,24 @@ func expire(addrs []DatabaseAddress, now int64) []DatabaseAddress {
 	}
 	return addrs
 }
+
+func sortedAddressCopy(addrs []DatabaseAddress) []DatabaseAddress {
+	sorted := make([]DatabaseAddress, len(addrs))
+	copy(sorted, addrs)
+	sort.Sort(databaseAddressOrder(sorted))
+	return sorted
+}
+
+type databaseAddressOrder []DatabaseAddress
```

Comment:
You don't need this anymore. There is sort.Sorted

Reply:
There is not? But if you mean `sort.SliceIsSorted` that takes a less func, I know, but then I'd need to repeat the less func closure in three places. This centralizes the sorting definition to one place.

Reply:
It needs to be a closure on the slice being sorted. Could be a function that takes the slice and returns the closure, but at that point I think a couple of one line methods to implement the interface is clearer.